cmd/rollout-operator/main.go

```diff
@@ -118,6 +120,17 @@ func main() {
 	ready := atomic.NewBool(false)
 	restart := make(chan string)
 
+	name := os.Getenv("JAEGER_SERVICE_NAME")
+	if name == "" {
+		name = "rollout-operator"
+	}
+
+	if trace, err := tracing.NewFromEnv(name); err != nil {
+		level.Error(logger).Log("msg", "Failed to setup tracing", "err", err.Error())
```

Comment:
Logging at error during start up but then continuing on seems odd to me. WDYT about logging at warning _or_ aborting if we can't set up tracing?

Reply:
Mimir does the same thing, but I've changed it in efae75d.